### PR TITLE
[BEAM-147] Adding sum of squares to distributions in metrics

### PR DESCRIPTION
**Pull request by pabloem**
Be sure to do all of the following to help us incorporate your contribution
quickly and easily:

 - [x] Make sure the PR title is formatted like:
   `[BEAM-<Jira issue #>] Description of pull request`
 - [x] Make sure tests pass via `mvn clean verify`. (Even better, enable
       Travis-CI on your fork and ensure the whole test matrix passes).
 - [x] Replace `<Jira issue #>` in the title with the actual Jira issue
       number, if there is one.
 - [ ] If this contribution is large, please file an Apache
       [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.txt).

---

### CODE REVIEW

**Comment by pabloem** [comment]
r: @bjchambers not many changes were added. Should be an easy review : )

File: sdks/python/apache_beam/metrics/cells_test.py [hunk 1]
```diff
@@ -86,35 +86,42 @@ def test_parallel_access(self):
...
 
-    total = (self.NUM_ITERATIONS * (self.NUM_ITERATIONS-1)/2 * self.NUM_THREADS)
+    total = ((self.NUM_ITERATIONS + 1) *
+             (self.NUM_ITERATIONS)/2 *
+             self.NUM_THREADS)
+
+    count = ((self.NUM_ITERATIONS + 1)* self.NUM_THREADS)
 
-    count = (self.NUM_ITERATIONS * self.NUM_THREADS)
+    squares = (self.NUM_ITERATIONS *
```
Comment (bjchambers) [inline sdks/python/apache_beam/metrics/cells_test.py]: Add a comment about this? At first glance it seems unrelated to the squares, so it would be good to explain why this is related.

Comment (pabloem) [inline sdks/python/apache_beam/metrics/cells_test.py]: Added an explanation for the test case, and the formulas.

**Comment by pabloem** [comment]
r: @robertwb if this looks fine to you, could you merge? : )

**Comment by robertwb** [comment]
This is an unstable algorithm: https://en.wikipedia.org/wiki/Algorithms_for_calculating_variance#Online_algorithm
